src/schema/v2/me/myBids.ts

```diff
@@ -379,3 +396,32 @@ function withSaleInfo(sale): SaleSortingInfo {
     liveBiddingStarted,
   }
 }
+
+function formatGravityLotStandingsToMatchCausalityLotStandings(
+  gravityLotStandings
+): LotStandingResponse {
+  return gravityLotStandings.map((lotStanding) => {
+    return {
+      lot: {
+        bidCount: lotStanding.sale_artwork.bidder_positions_count,
+        reserveStatus: lotStanding.sale_artwork.reserve_status,
+        internalID: lotStanding.id,
+        saleId: lotStanding.bidder.sale.id,
+        soldStatus: getSoldStatus(lotStanding),
+        sellingPriceCents: lotStanding.highest_bid_amount_cents,
+      },
+    }
+  })
+}
+
+function getSoldStatus(lotStanding): string {
+  const { sale_artwork } = lotStanding
+  const { artwork } = sale_artwork
+  if (artwork.forSale) {
+    return "ForSale"
+  } else if (artwork.sold) {
+    return "Sold"
+  } else {
+    return "Passed" // this is incorrect
```

Comment:
What does "passed" mean? Any idea where to find that from the lot standing?

Reply:
i _think_ that's something for live auctions so can probably skip/not worry about it here

Reply:
Passed is from live auction integrations, where the lot didn't sell.